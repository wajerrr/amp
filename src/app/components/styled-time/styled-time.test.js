import React from 'react';
import renderer from 'react-test-renderer';
import StyledTime from './styled-time';

describe('StyledTime', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(
        <StyledTime itemProp="datePublished" content="2018-04-16T07:01:00.000Z">
          Apr 16th 2018
        </StyledTime>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
