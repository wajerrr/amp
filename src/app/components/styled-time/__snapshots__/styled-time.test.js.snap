// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTime should match snapshot 1`] = `
<time
  content="2018-04-16T07:01:00.000Z"
  itemProp="datePublished"
>
  Apr 16th 2018
</time>
`;
